README.md

```diff
@@ -2,16 +2,24 @@
 
 ## Example Command
 
+[![build](https://github.com/KevenDuverglas/keven-cellveyor/actions/workflows/build.yml/badge.svg)](https://https://github.com/KevenDuverglas/keven-cellveyor/actions/workflows/build.yml)
+[![Language:Python](https://img.shields.io/badge/Language-Python-blue.svg)](https://github.com/KevenDuverglas/keven-cellveyor/search?l=python)
+[![Maintenance](https://img.shields.io/badge/Maintained%3F-Yes-blue.svg)](https://github.com/KevenDuverglas/keven-cellveyor/graphs/commit-activity)
+
 ```
 poetry run cellveyor --spreadsheet-directory \
-/home/gkapfham/working/data/gradebook/2023 --spreadsheet-file CMPSC-203-Fall-2023-Gradebook.xlsx \
+/home/gkapfham/working/data/gradebook/2023 --spreadsheet-file
+CMPSC-203-Fall-2023-Gradebook.xlsx \
```

Comment:
This should be further generalized.

It should not, ultimately, have anything to do with our class.